packages/block-editor/src/components/link-control/index.js

```diff
@@ -95,6 +95,7 @@ function LinkControl( {
 	value,
 	settings,
 	onChange = noop,
+	showSuggestions = true,
```

Comment:
Can we document this new prop, both in the associated JSDoc types definition and in the README.md?

Reply:
done.